src/Data/ServiceObject/ServiceObjectEditingConfig.ts

```diff
@@ -7,8 +7,8 @@ provideEditingConfig(ServiceObject, {
     isCarrier: { title: 'Is carrier?' },
     isCustomer: { title: 'Is customer?' },
     isSupplier: { title: 'Is supplier?' },
-    parentId: { title: 'Parent service object (ID)' },
+    parentId: { title: 'Parent service object ID' },
```

Comment:
* [ ] FYI some attribute names end in `Id`. With the new naming scheme we not longer explicitly highlight them.

Reply:
We will have do discuss this again, once we to have `reference` support.